src/Cake.Common/Tools/MSBuild/MSBuildRunner.cs

```diff
@@ -64,6 +64,13 @@ private ProcessArgumentBuilder GetArguments(FilePath solution, MSBuildSettings s
                 builder.Append(string.Concat("/p:\"Configuration\"=", configuration.Quote()));
             }
 
+            // Buid for a specific platform?
+            if (settings.PlatformTarget != default(PlatformTarget))
```

Comment:
I had same though originally, but explicitly specifying "AnyCPU" platform causes many other tests to fail (because before this change, AnyCPU was expected as default value) and I was too lazy to fix them. But, if you find this to be good idea as well, then it worth implementing :)